resources/metadata/helmExecute.yaml

```diff
@@ -336,6 +336,18 @@ spec:
           - PARAMETERS
           - STAGES
           - STEPS
+      - name: templateStartDelimiter
+        type: string
+        description: When templating value files, use this start delimiter.
+        default: "{{"
+        scope:
+          - STEPS
```

Comment:
we maybe need a `parameter` scope as well as it works with azure devops when directly interacting with the azure task

Reply:
Added to both parameters.